README.md

```diff
@@ -2,7 +2,7 @@
 
 > Warning: This project is currently being developed and the code shouldn't be used in production.
 
-## MS SQL Server - Amazon DMS Data Replication Demo
+## MS SQL Server - AWS DMS Data Replication Demo
```

Comment:
- Should we expand "DMS" here?
- Is it superfluous to have "Data"? ("Database Migration Service Data Replication Demo")

Reply:
done